modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -22,11 +22,7 @@
 sda.enkf.multisite <- function(settings, obs.mean, obs.cov, Q = NULL, restart=F, 
                                control=list(trace=T,
                                             FF=F,
-                                            interactivePlot=T,
-                                            TimeseriesPlot=T,
-                                            BiasPlot=F,
-                                            plot.title=NULL,
```

Comment:
`plot.title`  and `facet.plots` are arguments to `post.analysis.multisite.ggplot` and you used them, this would crash that function. Also you can use `TimeseriesPlot`  for what you did in line 472.

Reply:
i tried just turning off the plotting with the original parameters and it still kept failing at the plotting section of the function. I'm not planning on keeping that change, so I'll change it back for now, but something funky is going on with turning on/off the plotting parameters.

Reply:
fixed back to how it was